src/main/java/com/springboot/star_wars_character_popularity/app/exception/FileNotSavedException.java

```diff
@@ -0,0 +1,10 @@
+package com.springboot.star_wars_character_popularity.app.exception;
+
+import java.io.IOException;
+
+public class FileNotSavedException extends Exception{
```

Comment:
I am not sure, but I think that you should extend RuntimeException so your code doesn't break.